doc/source/ray-air/examples/pytorch_resnet_batch_prediction.ipynb

```diff
@@ -0,0 +1,398 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "2023-02-14 18:45:47,930\tINFO worker.py:1352 -- Connecting to existing Ray cluster at address: 10.0.49.247:6379...\n",
+      "2023-02-14 18:45:47,962\tINFO worker.py:1529 -- Connected to Ray cluster. View the dashboard at \u001b[1m\u001b[32m127.0.0.1:8265 \u001b[39m\u001b[22m\n",
+      "2023-02-14 18:45:48,239\tINFO packaging.py:373 -- Pushing file package 'gcs://_ray_pkg_30175d6ead5ce825f5779cb63cdb5463.zip' (90.97MiB) to Ray cluster...\n",
+      "2023-02-14 18:45:49,908\tINFO packaging.py:386 -- Successfully pushed file package 'gcs://_ray_pkg_30175d6ead5ce825f5779cb63cdb5463.zip'.\n"
+     ]
+    },
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "217255c5a2ba4ec5890f6f3667f5b429",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "VBox(children=(HTML(value='<h2>Dataset</h2>'), Tab(children=(HTML(value='<div class=\"scrollableTable jp-Render…"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import ray\n",
+    "from ray.data.datasource.partitioning import Partitioning\n",
+    "\n",
+    "s3_uri = \"s3://anonymous@air-example-data-2/imagenette2/val/\"\n",
+    "partitioning = Partitioning(\"dir\", field_names=[\"class\"], base_dir=s3_uri)\n",
+    "ds = ray.data.read_images(s3_uri, partitioning=partitioning, mode=\"RGB\")\n",
+    "ds\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "hide-cell",
+     "remove-cell"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "# If you want to run the full example, please set this to False\n",
+    "SMOKE_TEST = True"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "tags": [
+     "remove-cell"
+    ]
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Read progress: 100%|██████████| 50/50 [00:09<00:00,  5.42it/s]\n",
+      "Read progress: 100%|██████████| 50/50 [00:00<00:00, 3996.02it/s]\n"
+     ]
+    }
+   ],
+   "source": [
+    "#@title +\n",
+    "if SMOKE_TEST:\n",
+    "    ds = ds.limit(1000)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that the label for images are their corresponding directory names (e.g. n01728920). To find the indices of our model output that correspond to these names, we'll need to download a mapping from the s3 bucket (`imagenet_class_index.json`). The actual class names can also be found from this file."
```

Comment:
> Notice that the label for images are their corresponding directory names (e.g. n01728920).

Should we show a few rows of the dataset to make this more clear?

Reply:
The dataset is an image folder. After loading to ray dataset, each row will be like `{"image": np.array, "class": "n01728920"}`. Will it be so messy to print out the whole image tensors?

Reply:
Added a cell to show a input example.